charts/dapr/README.md

```diff
@@ -141,6 +141,7 @@ The Helm chart has the follow configuration options that can be supplied:
 | `dapr_placement.cluster.logStoreWinPath`       | Mount path for persistent volume for log store in windows when HA is true                                                                                                 | `C:\\raft-log`          |
 | `dapr_placement.volumeclaims.storageSize`      | Attached volume size                                                                                                                                                      | `1Gi`                   |
 | `dapr_placement.volumeclaims.storageClassName` | storage class name                                                                                                                                                        |                         |
+| `dapr_placement.maxApiLevel`                  | Sets the `max-api-level` flag which prevents the Actor API level for going above this value. The default value of -1 means no cap.  | `-1` |
```

Comment:
How about I add the more detailed explanation in the docs? I'd rather not write a long paragraph here.

Most users will not need to set this, but it can be necessary in some rare situations.

By default this is automatically set to the minimum observed value in the cluster. For example, say you have 3 apps, with these actor API levels: 10, 20, 20. The minimum observed is 10, so that will be the level used in the cluster.

Next, assume that the app with level 10 goes away, so the minimum level now becomes 20.

The level can never be decreased, only increased. Which means that if now the app with level 10 comes back, it will not be able to connect to Placement (and use actors).

In this very specific case, users can set `maxApiLevel` to `10` to enforce the reported level is never higher than 10.

I can see this possibly being the case during upgrades or if users have the need to run older versions of a sidecar alongside new ones. It should not be a common case, however.